python/runtime/utils/PyRestRemoteClient.cpp

```diff
@@ -424,7 +426,7 @@ class PyNvcfRuntimeClient : public PyRestRemoteClient {
       while (resultJs.contains("status") &&
              resultJs["status"] == "pending-evaluation") {
         const std::string reqId = resultJs["reqId"];
-        if (m_logLevel > LogLevel::None) {
+        if (m_logLevel > LogLevel::Info) {
```

Comment:
Note that this change means we won't get `Polling NVQC result data for Request Id` messages by default anymore.